### PR TITLE
Allows players to research and build Tug Upgrades, also Spellcheck

### PR DESCRIPTION
**Pull request by Bobbanz1**
## About The Pull Request
Allows people to now research those fighter tug upgrades! 
Military Police be aware for now those Janitors will be hunting down those who claim to be **The Drift King**

Also fixes the following Issues:
* Fixes https://github.com/BeeStation/NSV13/issues/2253
* Fixes https://github.com/BeeStation/NSV13/issues/1819

## Why It's Good For The Game
**_YOU KNOW THEY USED TO CALL ME THE DRIFT KING_**

Fixes a typo and actually lets you research things that were supposed to be researchable.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/59128051/210139016-e84c99b1-20d5-4d49-81f0-bce335ab7fba.mp4

</details>

## Changelog
:cl:
add: Added the Tug Upgrades as researchable things!
fix: Fixed a typo in the Light Fighter description
/:cl:

### CODE REVIEW

**Comment by Bobbanz1** [comment]
**GOD FUCKING DAMN IT ENTER KEY**

File: nsv13/code/modules/research/techweb/all_nsv_nodes.dm [hunk 1]
```diff
@@ -154,3 +154,48 @@
...
 	export_price = 5000
+
+//Vehicle Hardpoint tech
+/datum/techweb_node/vehicle_start
+	id = "vehicle_start"
+	display_name = "Mechanical Research"
+	description = "Researching the possibility of expanding the usage behind those useless Tugs"
+	prereq_ids = list("engineering")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
+	export_price = 1000
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/research/techweb/all_nsv_nodes.dm:165]: This should unlock the 'pathetic' tug engine IMO.

File: nsv13/code/modules/research/techweb/all_nsv_nodes.dm [hunk 2]
```diff
@@ -154,3 +154,48 @@
...
+	prereq_ids = list("vehicle_start")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2000)
+	export_price = 2000
+
+/datum/techweb_node/vehicle_tier1
+	id = "vehicle_tier1"
+	display_name = "Standard Vehicle Parts"
+	description = "Researching the strange spinning thingies found attached to the side of the Tug and what the loud rumbling box inside the Tug does."
+	design_ids = list("vehicle_engine", "vehicle_tyres")
+	prereq_ids = list("vehicle_utility")
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/research/techweb/all_nsv_nodes.dm:181]:
```suggestion
	prereq_ids = list("vehicle_start")
```
I feel like keeping utility and direct upgrades seperate paths, like with mecha equipment, is better.

Comment (Bobbanz1) [reply nsv13/code/modules/research/techweb/all_nsv_nodes.dm]: Done

File: nsv13/code/modules/research/techweb/all_nsv_nodes.dm (hunk 1 above)
Comment (Bobbanz1) [reply nsv13/code/modules/research/techweb/all_nsv_nodes.dm]: Altered it so this one unlocks the pathetic tug engine and the utilities don't

**Review by covertcorvid — APPROVED** [review]

**Pull request merged**